packages/core/test/specs/data_sources/model/data_collection/ComponentDataCollection.getters-setters.ts

```diff
@@ -40,37 +39,40 @@ describe('Collection component getters and setters', () => {
     let cmp: ComponentDataCollection;
 
     beforeEach(() => {
-      cmp = wrapper.components({
+      const cmpDef = {
         type: DataCollectionType,
-        collectionDef: {
-          componentDef: {
+        components: {
+          type: DataCollectionItemType,
+          components: {
             type: 'default',
             components: [
               {
                 type: 'default',
                 tagName: 'div',
                 attributes: {
                   dataUser: {
-                    type: DataCollectionVariableType,
-                    variableType: DataCollectionStateVariableType.currentItem,
+                    type: DataVariableType,
+                    variableType: DataCollectionStateType.currentItem,
                     collectionId: 'my_collection',
                     path: 'user',
                   },
                 },
               },
             ],
           },
-          collectionConfig: {
-            collectionId: 'my_collection',
-            startIndex: 1,
-            endIndex: 2,
-            dataSource: {
-              type: DataVariableType,
-              path: 'my_data_source_id',
-            },
+        },
+        collectionDef: {
```

Comment:
I guess `collectionDef` should be `dataResolver` to be consistent with variable/condition